documental/serializers.py

```diff
@@ -1,6 +1,7 @@
 from rest_framework import serializers
 
 from documental import models
+from django.conf import settings
```

Comment:
local imports, as `from documental import models` migth be at the end of imports.

Reply:
Removed.